### DISCUSSION TITLE
Reboot k3s agent without k3s server running

### DISCUSSION THREAD

**Discussion by fuch1m** [k3s-io/k3s#10685] [Q&A] [answered]
Hello, 

I have a question about wether it is possible to have a k3s server only online to change configuration of an agent and otherwise offline. I experiment and see that pods scheduled on an agent continue to run if I switch off the server. But if I reboot the agent the pods only get started again (restartPolicy: Always) if the server is online again. Is it possible to configure the agent to load its last configuration by its own without the server?

**Comment by shakibamoshiri** [comment]
good, question, hope someone tells yes and how

**Comment by brandond** [comment, marked as answer]
No, you cannot start or restart agents without a server available to validate the agent's configuration. If you had pods previously running before the restart, they will continue running, assuming they can tolerate the CNI and kube-proxy configuration potentially becoming out of sync with the actual state of the cluster.
